lib/api/api_connect.dart

```diff
@@ -7,24 +7,24 @@ import 'package:Solon/api/message.dart';
 import 'package:Solon/api/proposal.dart';
 // import 'package:Solon/api/comment.dart';
 // import 'package:Solon/api/event.dart';
-// import 'package:Solon/api/forumpost.dart';
 import 'package:Solon/api/vote.dart';
 import 'package:Solon/api/register.dart';
 import 'package:Solon/api/login.dart';
-// import 'package:Solon/api/encryptPassword.dart';
 import 'dart:io';
 import 'package:flutter/services.dart';
-// import 'package:intl/intl.dart';
-// import "package:pointycastle/export.dart";
-// import 'package:flutter_string_encryption/flutter_string_encryption.dart';
 
 class APIConnect {
   static String _url = "https://api.solonedu.com";
 
   //for StreamBuilder
   final StreamController<List<Proposal>> _proposals =
       StreamController<List<Proposal>>();
+
+  final StreamController<List<ForumPost>> _forumposts =
+      StreamController<List<ForumPost>>();
+
```

Comment:
these still throw the sink close thingy

Reply:
commented those out; they are unnecessary anyway